doc/src/pages/edit.tsx

```diff
@@ -20,18 +22,26 @@ const Edit: FC = () => {
     const path = getPath();
 
     setIsLoading(true);
+
+    if (path === '' || !path.includes('https://github.com/apache')) {
+      setPathExist(false);
+      setIsLoading(false);
+      return;
+    }
+
     fetch(path.replace('github.com', 'raw.githubusercontent.com').replace('/edit', ''))
       .then((res) => setPathExist(res.status !== 404))
       .finally(() => setIsLoading(false));
   }, []);
 
   const edit = useCallback(() => {
-    let path = getPath();
+    const path = getPath();
+
     if (!pathExist) {
-      const pathArr = path.replace('edit', 'new').split('/');
-      pathArr[pathArr.length - 1] = `?filename=${pathArr.at(-1)}`;
-      path = pathArr.join('/');
```

Comment:
Not sure if this is ever run but it seems like it is for creating a new file if doesn't exist?

Should we replace this logic?

Maybe instead we can add to this logic and set a new condition for an empty path. Because path does not exist and path is empty are different.

Again, I'm not sure if it is ever run so IDK if it would be any problem.

Reply:
Hi, It looks like the code isn't creating new files. 

I think it's attempting to replace the path, for example, from '/edit/#githubrepolink' to '/new/?filename=githubrepolink' if it doesn't exist. But, we're using getPath = () => window.location.hash.slice(1); to retrieve the path, which only gives us the hash - "githubrepolink'".

So, it seems that this logic isn't being utilized. Please correct me if I'm wrong. Thanks!

Reply:
I'm not sure, I will check with @SkyeYoung

Reply:
Sure, Thank you for your review, @pottekkat.

Reply:
This logic will actually jump to the github url of "Create a new file". The working logic is: The document site uses English as fallback, and when documents in Chinese or other languages ​​do not exist, they will automatically be displayed in English. But at this time, `edit url` will still use the path of the corresponding language. (For example, /xxx exists, but the document /zh/xxx does not exist. Assume that the edit url of the latter is github/zh/xxx. Since the document does not exist at this time, the content in /xxx will be displayed, but the edit url Still github/zh/xxx. Apparently at this point, at least I thought so, clicking this link should create this non-existent /zh/xxx).

Reply:
If you follow your modified logic, it should jump directly to a non-existent path, causing github to display 404.

So I think this current logic needs to be adjusted.

Reply:
Here is a practical example, please find the edit url link in the page 

https://apisix.apache.org/zh/docs/apisix/tutorials/manage-api-consumers/

Reply:
yes